new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js

```diff
@@ -58,29 +58,57 @@ const FiatBalance = ({ section, min = 0, max = 100, fieldWidth = 80 }) => {
       onReset={() => {
         setEditing(NAME, false)
       }}>
-      <Form className={classes.form}>
-        <PromptWhenDirty />
-        <Header
-          title="Cash out (Empty)"
-          editing={editing}
-          disabled={isDisabled(NAME)}
-          setEditing={it => setEditing(NAME, it)}
-        />
-        <div className={classes.wrapper}>
-          <div className={classes.first}>
+      {({ values }) => (
+        <Form className={classes.form}>
+          <PromptWhenDirty />
+          <Header
+            title="Cash out (Empty)"
+            editing={editing}
+            disabled={isDisabled(NAME)}
+            setEditing={it => setEditing(NAME, it)}
+          />
+          <div className={classes.wrapper}>
+            <div className={classes.first}>
+              <div className={classes.row}>
+                <Cashbox
+                  labelClassName={classes.cashboxLabel}
+                  emptyPartClassName={classes.cashboxEmptyPart}
+                  percent={
+                    values.fillingPercentageCassette1 ??
+                    data?.fillingPercentageCassette1
+                  }
+                  inFiatBalanceAlerts={true}
```

Comment:
This here. Also, I believe you can get away with omitting `={true}`, and just call `applyColorVariant` or whatever name you decide better fits the flag's purpose